### PR TITLE
Centralized location for Android versions

### PR DESCRIPTION
**Pull request by tjleing**
- Android version
- Kotlin version
- Geo version

only works on versions specified in code blocks

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/constants/versions.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  ANDROID_VERSION: "1.28.1",
+  KOTLIN_VERSION: "0.12.1",
```
Comment (changxu0306) [inline src/constants/versions.ts:3]:
```suggestion
  ANDROID_VERSION: "1.28.3",
  KOTLIN_VERSION: "0.12.3",
```

File: src/constants/versions.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  GEO_VERSION: "0.1.0",
```
Comment (changxu0306) [inline src/constants/versions.ts:4]: Is the geo version shared across platforms?

File: src/constants/versions.ts (hunk 1 above)
Comment (changxu0306) [reply src/constants/versions.ts:3]: Maybe `ANDROID_KOTLIN_VERSION` is a better name? `KOTLIN_VERSION` sounds like the version of Kotlin language.

File: src/constants/versions.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  ANDROID_VERSION: "1.28.1",
+  ANDROID_KOTLIN_VERSION: "0.12.1",
```
Comment (tjleing) [inline src/constants/versions.ts:3]:
```suggestion
  ANDROID_VERSION: "1.28.3",
  ANDROID_KOTLIN_VERSION: "0.12.3",
```

**Review by changxu0306 — APPROVED** [review]
LGTM! Thank you @tjleing 😊

**Review by siegerts — APPROVED** [review]

**Pull request merged**